File: Android/HatPepper_Java/app/src/main/java/jp/nuits/hatpepper/MainActivity.java
package jp.nuits.hatpepper;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import jp.nuits.hatpepper.presentations.RestaurantsFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, RestaurantsFragment.newInstance())
                    .commitNow();
        }
    }
}

File: Android/01.Before/app/src/main/java/jp/nuits/hatpepper/MainActivity.kt
package jp.nuits.hatpepper

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import jp.nuits.hatpepper.presentation.NearbyRestaurantsFragment


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .replace(R.id.container, NearbyRestaurantsFragment.newInstance())
                .commitNow()
        }
    }
}
